build-files/hocort/1.2.2/README.md

```diff
@@ -0,0 +1,42 @@
+# HoCoRT
+
+Main tool: [HoCoRT](https://github.com/ignasrum/hocort)
+  
```

Comment:
There are several additional tools that are installed in this image.

Can you list them with their respective versions?